src/Studio/Plugins/Graph/GraphAttributesWidget.h

```diff
@@ -114,17 +118,41 @@ class GraphAttributesWidget : public QScrollArea, public Core::EventHandler
 
 		void AddConditions(GraphObject* object, bool readOnly);
 		void AddActions(GraphObject* object, bool readOnly);
-
+        void dataReceived(QByteArray data);
+        void On_connect();
+        void On_Ok();
 		uint32 FindRemoveButtonIndex(QObject* button) const;
 
 		GraphObject*					mGraphObject;
 		GraphObjectViewWidget*			mGraphObjectView;
 		PropertyTreeWidget*				mPropertyTreeWidget;
 		Property*						mNameProperty;
+		Property*						mNameProperty_1;
```

Comment:
Please init these new pointer fields to null in the constructor like done for `mNameProperty`.
Please also double check it in other classes.